src/TextForEvent.js

```diff
@@ -366,6 +366,15 @@ function textForCallInviteEvent(event) {
 
 function textForThreePidInviteEvent(event) {
     const senderName = event.sender ? event.sender.name : event.getSender();
+
+    if (!event.getContent().display_name) {
```

Comment:
There are others available, but this one seemed like a good victim for what we're doing. Elsewhere we do check all 4 required properties - what are your thoughts here?

Reply:
I think it's okay as long as we make it clear to future readers. The condition seems surprising at a glance, so a comment should be enough I think.

I guess we could also extract this block:

https://github.com/matrix-org/matrix-react-sdk/blob/3cd4b6992d85be8b34a76bb5212d4ce3e04978c9/src/components/views/rooms/MemberList.js#L375-L379

to some kind of shared helper, like `InviteUtils.isValidInvite` so that the purpose would be clear without needing comments per se, but maybe that's overkill.